app/ios/Podfile.lock

```diff
@@ -980,4 +980,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: f043d17a36d946983917d5651311899450c41e12
 
-COCOAPODS: 1.13.0
+COCOAPODS: 1.15.2
```

Comment:
Should update with `yarn ios:setup` from `app/` so it uses the Gemfile and locked versions. Cocoapods version should be the same

Reply:
Rather than just `pod install` I mean